en/docs/administer/managing-users-and-roles/managing-user-roles.md

```diff
@@ -108,15 +108,15 @@ Follow the instructions below to create the `creator` , `publisher` and `subscri
 
             By default, the scope-role mapping contains Internal/creator, Internal/publisher, Internal/subscriber as the default roles. If there are custom roles defined with API creator, API publisher, admin and API subscriber permissions, those roles have to be configured in `tenant-conf.json` under relevant scopes.
 
-7. Log in to admin portal ( `https://<APIM_Host>:<APIM_Port>/admin`) and navigate to Settings > Scope Mapping in admin portal. 
+7. Log in to admin portal ( `https://<APIM_Host>:<APIM_Port>/admin`) and navigate to Settings > Role Permissions in admin portal. 
 
 8. Update mappings and add the new role created above(creator) under Roles of all scopes that it should be assigned to as follows.  
 
     **For example**, if the new role has the API Creator permission, add it under every scope that has `Internal/creator` role specified.
 
-    If you create a custom role that has different permissions, add that role under the required scopes based on the functionality or permissions you need to give to a user carrying this role. For example, if you need to allow the user to create apis, add the new role under `apim:api_create` scope.
+    If you create a custom role that has different permissions, add that role under the required scopes based on the functionality or permissions you need to give to a user carrying this role. For example, if you need to allow the user to create apis, click the new scope `apim:api_create` under **custom permissions**.
```

Comment:
if we have role mapping.. do we need to do this? can't we do role mapping instead?

Reply:
Yes, it will separately add the required scopes to the role.  https://github.com/wso2/docs-apim/pull/1695/files#diff-d9a69ced0da39084ee9b8f744f06e17bR176